### PR TITLE
Add support for browser options in testing

### PR DESCRIPTION
**Pull request by joshuarmil**

### CODE REVIEW

**Comment by joshuarmil** [comment]
Hmmm, I added the comments to you in the description of the commits but maybe I should have just commented here? Doesn't show the descriptions in this log

**Comment by joshuarmil** [comment]
Also, do I have to do something to cancel the previous commit that you wanted changes for, since it's superseded  by the new one?

**Comment by ptravers** [comment]
No that's a good thing to keep it shows the reason I wanted it changed and why and so forth. Git is as much about history of construction as about controlling the source in a safe way.

**Comment by ptravers** [comment]
Normally you would add them here but I can see them now.

File: selenium-tests/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "test_homepage_exists.js",
   "scripts": {
     "test": "node test_homepage_exists.js"
```
Comment (ptravers) [inline selenium-tests/package.json]: you're missing a comma at the end of this line I think

File: autotest.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+while true; do
+	result=$(curl -g http://localhost:3000/status)
+	if [ "$result" == "UP" ]; then
+		echo "Server UP"
+		break
+	fi
+done
+node selenium-tests/test_homepage_exists_wsl.js
+pnode=$(pidof node)
+kill -9 $pnode
```
Comment (ptravers) [inline autotest.sh]: this is fine I think.

File: selenium-tests/test_homepage_exists_wsl.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                       Assert(expectedMessage == text, testMessage);
+		    });
+    } catch(error) {
+      Assert.fail(error.message);
+    } finally {
+      await driver.quit();
+    }
+  });
+
+
```
Comment (ptravers) [inline selenium-tests/test_homepage_exists_wsl.js]: Just delete the old file then. Not totally sure I understand the issue?

**Comment by ptravers** [comment]
I'm happy to help you switch over if you need it. is there anything you do in Windows that you need to keep? like do you have some Windows software you need?

File: selenium-tests/package.json (hunk 1 above)
Comment (joshuarmil) [reply selenium-tests/package.json]: yes you are right, I will add

**Comment by ptravers** [comment]
I will test it later today or tomorrow

**Comment by joshuarmil** [comment]
@ptravers Was struggling to get it to work on ubuntu until I realized chromium doesn't just take the place of chrome without modifications, so i downloaded chrome, works now, so no need for both files. only thing is, the bash script doesn't work as the bin/www file isn't implemented in this branch, and neither is the status route, but should be good once everything is merged. as is though, probably wouldn't work with travis

**Comment by ptravers** [comment]
awesome! what's left to do for the status route PR?

**Comment by joshuarmil** [comment]
@ptravers I think it should be good, I was just waiting for the request to be approved, but it works well on my end

**Comment by joshuarmil** [comment]
Is there a specific way I should be declaring the changes met, or should I be like, reissuing the pull request, or do i just add to it like i've been doing with additional commits?

**Comment by ptravers** [comment]
nah that's all on me for not approving. sometimes you have to poke your reviewers either here or by text

**Pull request merged**